src/expr/type_checker_template.cpp

```diff
@@ -1,3 +1,36 @@
+/******************************************************************************
```

Comment:
I don't think this change should have happened, this header is only supposed to be added to the generated files in the build directory, right?